FE/todo-app/src/components/loginForm/LoginForm.tsx

```diff
@@ -35,45 +35,24 @@ export const LoginForm = () => {
         <div className="border-b  mx-10">
           <div className="flex justify-center px-4 py-2">Log to ToDo app</div>
         </div>
-        <form onSubmit={handleSubmit(onSubmit)}>
-          <div className="pb-5 pt-10 flex justify-center ">
-            <input
-              type="text"
-              className="bg-black px-3 py-1 rounded-md text-green-500"
-              placeholder="Username"
-              {...register("identifier", {
-                required: "Username is required",
-              })}
-            />
-          </div>
-          <div className="flex justify-center text-red-500">
-            <p>{errors.identifier?.message}</p>
-          </div>
+        <FormProvider {...methods}>
+          <form onSubmit={methods.handleSubmit(onSubmit)}>
+            <Input type="text" placeholder="Username" name="identifier" description="Username" />
+            <Error errorMsg={methods.formState.errors.identifier?.message} />
 
-          <div className="p-5 flex justify-center ">
-            <input
-              type="password"
-              className="bg-black px-3 py-1 rounded-md text-green-500"
-              placeholder="Password"
-              {...register("password", {
-                required: "password is required",
-              })}
-            />
-          </div>
-          <div className="pb-5 flex justify-center text-red-500">
-            <p>{errors.password?.message}</p>
-          </div>
-          <button className={styles.loginButton} type="submit">
-            Login
-          </button>
-          <p className="text-red-500 flex justify-center">{isError && "wrong username or password"}</p>
-          <div className="border-t  mx-10">
-            <div className="flex justify-center px-4 py-2">Not account yet? Let&lsquo;s create one</div>
-          </div>
-        </form>
-        <button className={styles.loginButton} type="submit" onClick={() => router.push("/registration")}>
-          Registration
-        </button>
+            <Input type="password" placeholder="Password" name="password" description="Password" />
+            <Error errorMsg={methods.formState.errors.password?.message} />
+
+            <Button label="Login" buttonType="submitType" />
+            {isError && <Error errorMsg="wrong username or password" />}
+
+            <div className="border-t  mx-10">
+              <div className="flex justify-center px-4 py-2">Not account yet? Let&lsquo;s create one</div>
+            </div>
```

Comment:
co je tohle zač? nemělo by to být taky specialní komponenta? A proč ten text nemáš třeba v <p>? nebo něco takového?